src/module/actor/actor-sheet.js

```diff
@@ -108,7 +108,7 @@ export class OseActorSheet extends ActorSheet {
     if (item.type === "container" && itemData.itemIds) {
       const containedItems = itemData.itemIds;
       const updateData = containedItems.reduce((acc, val) => {
-        acc.push({ _id: val.id, "system.containerId": "" });
+        acc.push({ _id: val, "system.containerId": "" });
```

Comment:
You're addressing this in #299, aren't you?

Reply:
> You're addressing this in #299, aren't you?

I am, yes! I needed the change for that, so I cherry-picked it into that branch.